README.md

```diff
@@ -63,6 +63,8 @@ $ nvs use chakracore-nightly
 
 ### Building Node-ChakraCore
 
+Node-ChakraCore can be build on Wndows / Linux / Mac OS.
```

Comment:
Windows needs to be spelled correctly. Also, this is a nitpick, but I feel like if we are going to say this, we should say exactly what kinds of linux and possibly minimum versions numbers for all three OSes.

Reply:
I don't feel like this is necessary.  The sections below fulfill the same purpose.